### PR TITLE
doc: add ios swift sdk

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/sdks/ios-swift/index.md [hunk 1]
```diff
@@ -0,0 +1,484 @@
+---
+title: iOS SDK
```
Comment (qingzhuozhen) [inline docs/data/sdks/ios-swift/index.md:2]: Add (Beta) in title?

File: docs/data/sdks/ios-swift/index.md [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+    ```bash
+    github "amplitude/Amplitude-Swift" ~> 0.3.0
+    ```
+
+
+### Initialization
+
+Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. You can use the iOS SDK anywhere after it's initialized in an iOS application.
+
+```swift
```
Comment (qingzhuozhen) [inline docs/data/sdks/ios-swift/index.md:55]: Ideally we can have obj-c as well, I am fine to start with swift now

File: docs/data/sdks/ios/index.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ icon: simple/ios
 This is the official documentation for the Amplitude Analytics iOS SDK.
 
 !!!info "iOS SDK Resources"
-    [:material-github: GitHub](https://github.com/amplitude/Amplitude-iOS) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-iOS/releases) · [:material-book: API Reference](http://amplitude.github.io/Amplitude-iOS)
+    [:material-github: GitHub](https://github.com/amplitude/Amplitude-Swift) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-Swift/releases) · [:material-book: API Reference](http://amplitude.github.io/Amplitude-Swift)
```
Comment (qingzhuozhen) [inline docs/data/sdks/ios/index.md:13]: This is for legacy iOS right? We don't need to change the link here.

File: docs/data/sdks/ios/index.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
-title: iOS SDK
-description: The Amplitude iOS SDK installation and quick start guide. 
+title: iOS SDK (Legacy)
+description: The Amplitude iOS Swift SDK installation and quick start guide.
```
Comment (qingzhuozhen) [inline docs/data/sdks/ios/index.md:3]: We don't need Swift on legacy SDK

File: docs/data/sdks/ios-swift/index.md (hunk 1 above)
Comment (liuyang1520) [reply docs/data/sdks/ios-swift/index.md]: Thanks for reviewing! Good catch, updated in latest commit.

File: docs/data/sdks/ios-swift/index.md (hunk 2 above)
Comment (liuyang1520) [reply docs/data/sdks/ios-swift/index.md:55]: Agree with the obj-c examples, I will create a ticket to track it, and start with swift for now.

**Review by qingzhuozhen — APPROVED** [review]

**Pull request merged**